pkgs/applications/networking/p2p/transgui/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv, fetchsvn, pkgconfig, makeDesktopItem, unzip, fpc, lazarus,
+libX11, glib, gtk, gdk_pixbuf, pango, atk, cairo }:
+
+stdenv.mkDerivation rec {
+  name = "transgui-trunk-r${version}";
+  version = "986";
+  
+  src = fetchsvn {
+    url = "https://svn.code.sf.net/p/transgui/code/trunk/";
+    rev = "${version}";
+    sha256 = "0z83hvlhllm6p1z4gkcfi1x3akgn2xkssnfhwp74qynb0n5362pi";
+  };
+
+  buildInputs = [ pkgconfig unzip fpc lazarus stdenv.cc libX11
+                  glib gtk gdk_pixbuf pango atk cairo
+                ];
+                
+  makeFlags = [
+    "FPC=fpc"
+    "PP=fpc"
+    "INSTALL_PREFIX=$(out)"
+    "LCL_PLATFORM=gtk2"
+  ];
+
+ desktopItem = makeDesktopItem rec {
+    name = "transgui";
+    exec = name;
+    icon = name;
+    comment = meta.description;
+    desktopName = "Transmission Remote GUI";
+    genericName = "BitTorrent Client";
+    categories = "Application;Network;FileTransfer;P2P;GTK;";
+    mimeType = stdenv.lib.concatStringsSep ";" [
+      "application/x-bittorrent" "x-scheme-handler/magnet"
+    ];
+  };
+  
+  preBuild = ''
+    export makeFlags="$makeFlags INSTALL_PREFIX=$out/"
+    export NIX_LDFLAGS="$NIX_LDFLAGS -L${stdenv.cc.cc.lib}/lib \
```

Comment:
This can be set via a `NIX_LDFLAGS` attribute (all attributes are simply exported to the builder as an environment variable of the same name).

Also, does the build really not set the appropriate linker flags by itself?